packages/clerk-js/webpack.config.js

```diff
@@ -170,22 +170,58 @@ const typescriptLoaderDev = () => {
   };
 };
 
-/** @type { () => (import('webpack').RuleSetRule) } */
+/**
+ * Used in outputs that utilize chunking, and returns a URL to the stylesheet.
+ * @type { () => (import('webpack').RuleSetRule) }
+ */
 const clerkUICSSLoader = () => {
```

Comment:
These could be one loader with a boolean argument for `supportsChunks`. I didn't have a strong opinion one way to the other, so happy to change this if anyone else has a stronger opinion!